docs/development.md

```diff
@@ -161,7 +184,35 @@ Start in the GP2040-CE folder. **From a Developer Powershell or Developer Comman
     ```
 
 4. Your UF2 file should be in the build directory.
-
+### Windows (Pico SDK)
```

Comment:
Space above here

Reply:
Do you want me to add space? It's a bit unclear.